packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -284,34 +284,34 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	/**
 	 * {@link StarkActionBarConfig} object for the action bar component in regular mode
 	 */
-	public customTableRegularActions: StarkActionBarConfig;
+	public customTableRegularActions: StarkActionBarConfig = { actions: [] };
 
 	/**
 	 * MatTableDataSource associated to the MatTable embedded in this component
 	 */
-	public dataSource: MatTableDataSource<object>;
+	public dataSource!: MatTableDataSource<object>;
 
 	/**
 	 * Array of columns (column id's) to be displayed in the table.
 	 */
 	public displayedColumns: string[] = [];
 
-	public _globalFilterFormCtrl: FormControl;
+	public _globalFilterFormCtrl: FormControl = new FormControl();
```

Comment:
Same remark: we should use the `!` modifier to avoid creating a dummy formControl which will never be used

Reply:
I resolved this similarly to how we discussed the previous comment. 😄